modules/loader/src/test/java/com/opengamma/strata/loader/csv/TradeCsvLoaderTest.java

```diff
@@ -599,7 +600,9 @@ public void test_load_swap_full5() {
                     .startDate(date(2017, 8, 1))
                     .endDate(date(2022, 8, 1))
                     .frequency(Frequency.P3M)
-                    .businessDayAdjustment(BusinessDayAdjustment.NONE)
+                    .businessDayAdjustment(BusinessDayAdjustment.of(
```

Comment:
Is it still possible to get `NONE`? Is there a test for that? I suspect the code doesn't allow for it.

Reply:
No longer possible to get NONE for BusinessDayAdjustment. There should always be a default from now on.